src/MuTalk-Tests/MutationTestingFixedNumberOfMutantsBudgetTest.class.st

```diff
@@ -0,0 +1,51 @@
+Class {
+	#name : 'MutationTestingFixedNumberOfMutantsBudgetTest',
+	#superclass : 'MutationTestingBudgetTest',
+	#category : 'MuTalk-Tests',
+	#package : 'MuTalk-Tests'
+}
+
+{ #category : 'running' }
+MutationTestingFixedNumberOfMutantsBudgetTest >> runAnalysisFor: aNumber [
+
+	^ self
+		  runAnalysisWithBudget: (MutationTestingFixedNumberOfMutantsBudget for: aNumber)
+		  on: { AuxiliarClassForMutationTestingAnalysis }
+		  withTests: { AuxiliarClassForMutationTestingAnalysisTest }
+]
+
+{ #category : 'tests' }
+MutationTestingFixedNumberOfMutantsBudgetTest >> testEvaluateAllMutantsWithConstraintGreaterThanActualNumberOfMutations [
+
+	self runAnalysisFor: 50.
```

Comment:
Could we add a comment? It's not **_really_** clear that the analysis has 25 mutations, and that could change depending on the tested code, right?

Reply:
Same in all other methods in the class?